util/fibers/detail/scheduler.cc

```diff
@@ -638,6 +638,31 @@ void Scheduler::RunDeferred() {
 #endif
 }
 
+void Scheduler::PrintAllFiberStackTraces() {
+  ProcessRemoteReady();
```

Comment:
Why are you calling this here?

Reply:
`ProcessRemoteReady()` is pushing fibers into the ready queue if they were scheduled from `ScheduleFromRemote`. The remote queue doesn't support iteration and it's emptied into `ready_queue_` anyway.

Reply:
so please add a comment saying that you call it to gather more fibers that are in the process of being activated into ready queue.

Reply:
Since we now just loop over the fibers, this doesn't seem to matter too much anymore so I removed it.